e2e-tests/test_last_state_upgrade.sh

```diff
@@ -44,6 +44,17 @@ function wait_for_jobmanager_running() {
     wait_for_logs $jm_pod_name "Rest endpoint listening at" ${TIMEOUT} || exit 1
 }
 
+function assert_available_slots() {
+  expected=$1
+  ip=$(minikube ip)
+  actual=$(curl http://$ip/default/${CLUSTER_ID}/overview 2>/dev/null | grep -E -o '"slots-available":[0-9]+' | awk -F':' '{print $2}')
```

Comment:
wouldnt slots available depend on TM config only? that used to work even before I think

Reply:
I do not think so. If the parallelism is configured 2, we will get 0 available slot after the job is running.

Reply:
ah okay, I misunderstood the check. good point

Reply:
I think you mean the "slots-total" :)

Reply:
yep